inst/doc/ValidateModel.Rmd

```diff
@@ -107,8 +132,8 @@ colnames(failures) <- c("Commodity", "")
 reportValidationResult(failures)
 ```
 
-#### 5. Sum of each commodity's output across all states must almost equal (<= 1.11^7, or $11.1 million by commodity) commodity output in US Use Table minus International Imports (commodity specific). Even if the threshold is met, track the difference for each commodity. Save result as a type of quality control check
-```{r check5}
+#### 6. Sum of each commodity's output across all states must almost equal (<= 1.11^7, or $11.1 million by commodity) commodity output in US Use Table minus International Imports (commodity specific). Even if the threshold is met, track the difference for each commodity. Save result as a type of quality control check
+```{r check6}
```

Comment:
Rethinking about this validation check. See reasoning above.

Reply:
Looks good i'm interested in seeing the output to see what the diff is

Reply:
I modified the `df0` in [this check](https://github.com/USEPA/stateior/commit/25e8d6984e4c79e87413d5f23f80dde62ee2066a#diff-5df2e7e5c98e0c9c81c4c6cfcdc1c30fd768e667db1393a82ce2b54bf5c649baR141). Now the diff should be within $10M.

Reply:
@WesIngwersen check 6 is [updated](https://github.com/USEPA/stateior/blob/dev/inst/doc/output/ValidateModel_2012.md#6-sum-of-each-commoditys-output-across-all-states-must-almost-equal--1e7-or-10-million-by-commodity-commodity-output-in-us-use-table-minus-international-imports-commodity-specific) to show the difference even if there is no failure.